### PR TITLE
adds CR playbook

### LINKED ISSUE #77: [Playbook]: Costa Rica (CR)
### Country

Costa Rica (CR)

### eAIP

_No response_

### Steps to add a new playbook

You can check [playbooks](https://github.com/drph4nt0m/avbot-charts/tree/main/playbooks) for examples on chart playbooks and [axios engine](https://github.com/drph4nt0m/avbot-charts/blob/main/app/engines/axios.js) on how the features in the playbook are used by the axios scraping engine.

Tasks summary:
-[ ] Fork & clone AvBot Charts
-[ ] Create playbook for the country in playbooks
-[ ] Add tests for the country in tests/getChart.test.js (tests should be in alphabetical order of the country code)
-[ ] Push changes and submit a pull request
-[ ] If you need any help, reach out to us on our [Discord server](http://link.avbot.in/support).

Are you ready to work on this issue? Let us know, and we will assign it to you.

Happy Coding!

### PR DESCRIPTION
**Pull request by dmarkowicz**
# Description

Adds Costa Rica (CR) playbook. This is conventional AIP AD 2 in pdf, no eAIP available.
I also added lower case icao to template variables.

Closes #77

## How Has This Been Tested

Added tests for the country in tests/getChart.test.js

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation/readme
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/engines/axios.js [hunk 1]
```diff
@@ -46,18 +47,18 @@ module.exports = async (features, icao) => {
       }
     }
     if (features.chart) {
-      const lnk = $(fmtr(features.chart.selector, { icao })).attr(features.chart.attribute);
+      const lnk = $(fmtr(features.chart.selector, variables)).attr(features.chart.attribute);
       if (!lnk) {
         throw new Error('Not Found');
       }
       return `${fmtr(features.chart.baseUrl, { baseUrl: features.baseUrl })}${lnk.replaceAll(' ', '%20')}`;
```
Comment (drph4nt0m) [inline app/engines/axios.js:54]: Can you also move `baseUrl` to the `variables` and maybe rename the `variables` to `fmtrOptions`.

**Review by drph4nt0m — CHANGES_REQUESTED** [review]
@dmarkowicz this is a nice idea. I never thought of this. Thanks for it. Can you also make the requested tweaks? 
If you want, I can also create a new issue just for changes to `axios.js`.

File: app/engines/axios.js (hunk 1 above)
Comment (dmarkowicz) [reply app/engines/axios.js:54]: Done.

Comment (drph4nt0m) [reply app/engines/axios.js:54]: Line 54: should pass `fmtOptions` instead of `{ baseUrl: features.baseUrl }` as the second argument to `fmtr`

**Review by drph4nt0m — CHANGES_REQUESTED** [review]

File: app/engines/axios.js (hunk 1 above)
Comment (dmarkowicz) [reply app/engines/axios.js]: Right, I missed it.

**Review by drph4nt0m — APPROVED** [review]

**Comment by drph4nt0m** [comment]
@dmarkowicz thanks for your contribution. Looking forward to more PRs from you. I have also added few non playbook issues. Check them if you are interested.

**Pull request merged**